orchestrator/api/api_v1/endpoints/subscriptions.py

```diff
@@ -120,6 +122,25 @@ def in_use_by_subscriptions(subscription_id: UUID) -> List[SubscriptionTable]:
     return query_in_use_by_subscriptions(subscription_id).all()
 
 
+@router.post("/subscriptions_for_in_used_by_ids", response_model=Dict[UUID, SubscriptionSchema])
+def subscriptions_by_in_used_by_ids(data: List[UUID] = Body(...)) -> Dict[UUID, SubscriptionSchema]:
+    res = db.session.execute(
+        select(SubscriptionInstanceTable.subscription_instance_id, SubscriptionTable)
+            .filter(
+            SubscriptionTable.subscription_id.in_(
+                select(SubscriptionInstanceTable.subscription_id)
+                    .filter(SubscriptionInstanceTable.subscription_instance_id.in_(data))
+                    .subquery()
+            )
+        )
+
+    ).all()
+    result = {row[0]: row[1] for row in res if row[0] in data}
```

Comment:
@Mark90 could you have a look at this?

During the tests it seems to contain all instance_id's : so the filter is needed. Not sure which solution is the most efficient. (that seems to depend on amount of subs)

Reply:
(from our slack chat) My previously suggested query was indeed borked. This one does what it should:

```python
res = db.session.execute(
  select(SubscriptionInstanceTable)
  .join(SubscriptionTable)
  .filter(SubscriptionInstanceTable.subscription_instance_id.in_(data))
).all()
return {str(row[0].subscription_instance_id): row[0].subscription for row in res}
```

Reply:
Implemented that one. Thanks!